Documentation/improvements.md

```diff
@@ -0,0 +1,41 @@
+
+[Point.ToVector2()](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Point_ToVector2) - this is easy to use conversion from [Point](http://www.monogame.net/documentation/?page=T_Microsoft_Xna_Framework_Point) to [Vector2](http://www.monogame.net/documentation/?page=T_Microsoft_Xna_Framework_Vector2). Use it like this 
+
+    Point point = new Point(10,10);
+    var vector2 = point.ToVector2();
+
+[Rectangle.Contains(float,float)](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Rectangle_Contains_1) - float-based version of [Rectangle.Contains(int,int)](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Rectangle_Contains)
+
+[Rectangle.Contains(Vector2)](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Rectangle_Contains_4) - float-based version of [Rectangle.Contains(Point)](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Rectangle_Contains_2)
+
+[Rectangle.Contains(ref Vector2,out bool)](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Rectangle_Contains_5) - float-based version of [Rectangle.Contains(ref Point,out bool)](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Rectangle_Contains_3)
+
+[Rectangle.Offset(float,float)](http://www.monogame.net/docs/html/M_Microsoft_Xna_Framework_Rectangle_Offset_3.html) - float-based version of [Rectangle.Offset(int,int)](http://www.monogame.net/docs/html/M_Microsoft_Xna_Framework_Rectangle_Offset.html)
+
+[Rectangle.Offset(Vector2)](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Rectangle_Offset_1) - float-based version of [Rectangle.Offset(Point)](http://www.monogame.net/docs/html/M_Microsoft_Xna_Framework_Rectangle_Offset_2.html)
+
+[Rectangle.Size](http://www.monogame.net/documentation/?page=P_Microsoft_Xna_Framework_Rectangle_Size) - this property is a [Point](http://www.monogame.net/documentation/?page=T_Microsoft_Xna_Framework_Point) which contains width and height of this rectangle.
+
+[Vector2.ToPoint()](http://www.monogame.net/documentation/?page=M_Microsoft_Xna_Framework_Vector2_ToPoint) - this is easy to use conversion from [Vector2](http://www.monogame.net/documentation/?page=T_Microsoft_Xna_Framework_Vector2) to [Point](http://www.monogame.net/documentation/?page=T_Microsoft_Xna_Framework_Point). Use it like this 
+
+    Vector2 vector2 = new Vector2(10.01f,10.99f);
+    var point = vector2.ToPoint(); // point will be 10,10. 
+
+##Input
+
+[MouseState.Position](http://www.monogame.net/documentation/?page=P_Microsoft_Xna_Framework_Input_MouseState_Position) - property which returns a [Point](http://www.monogame.net/documentation/?page=T_Microsoft_Xna_Framework_Point) which contains X and Y parameters of MouseState. Before this change it is harder to obtain mouse position in a good way. You can now write something like this
+
+    spriteBatch.Draw(texture,mouseState.Position.ToVector2(),Color.White);
```

Comment:
Same thing....  Add a colon at the end of the sentence before it.... make a code block using ` ````` `.